.github/workflows/ci-latest-release.yml

```diff
@@ -116,9 +199,9 @@ jobs:
 
       - name: Sign the Container Images
         run: |
-          cosign sign -r kubearmor/kubearmor@${{ steps.digest.outputs.imagedigest }} --yes
-          cosign sign -r kubearmor/kubearmor-init@${{ steps.digest.outputs.initdigest }} --yes
-          cosign sign -r kubearmor/kubearmor-ubi@${{ steps.digest.outputs.ubidigest }} --yes
+          cosign sign -r public.ecr.aws/k9v9d5v2/kubearmor/kubearmor@${{ steps.digest.outputs.imagedigest }} --yes
```

Comment:
@sanjay-ba let's keep images in docker registry as well. use regctl to copy images in ECR registry.